phoenicis-tools/src/main/java/com/playonlinux/tools/files/FileUtilities.java

```diff
@@ -80,6 +82,14 @@ public String getFileContent(File file) throws IOException {
         return FileUtils.readFileToString(file, "UTF-8");
     }
 
+    public long getSize(File file) throws IOException {
+        Path folder = Paths.get(file.getAbsolutePath());
```

Comment:
assertI**n**Directory will throw an exception if the file is not in PlayOnLinux root